packages/components/src/resizable-box/style.scss

```diff
@@ -0,0 +1,43 @@
+.components-resizable-box__handle {
+	display: none;
+
+	// Show the resize handle when selected.
+	.components-resizable-box__container.is-selected & {
```

Comment:
Where do we document how a resizable box becomes "selected" to unhide itself? The current documentation for `ResizableBox` has no indication of this default hidden state. It was quite the journey to find my way here, as I've found it quite difficult to use the component.

Reply:
The behavior here was inherited from the original, non-componentized usages of ResizableBox; you can see it in previous PRs as well such as [here](https://github.com/WordPress/gutenberg/pull/10331/files#diff-01bcf979fbfe3fb1e688ddcddcbedd66).

There are probably other use-cases that would provide more valuable context, and could certainly be referenced and improved in the documentation.

Reply:
While I'd agree it often needs contextualized styling, the main problem for me was that in using the example from the documentation, there is nothing visual which appears. It took me quite a while to realize that the handles are hidden by default.

I understand this is largely a result of how it was used prior to this pull request, but it seems to me that as a generic component, there's no reason to hide those controls by default. I could imagine we might want to keep this behavior for blocks in a block list, but rather than styling in ResizableBox and in individual block stylesheets, it seems like something which should be managed in one of the stylesheets for the block list (apply consistently and exclusively to ResizableBox within blocks).